### PR TITLE
workflow: trigger nydus test in workflow

### PR DESCRIPTION
**Pull request by ChengyuZhu6**
trigger nydus test in workflow.

### CODE REVIEW

**Comment by Xynnn007** [comment]
I just retriggered the CI to check if it is caused by network fluctuations

**Review by Xynnn007 — APPROVED** [review]
lgtm

File: .github/workflows/image_rs_build.yml [hunk 1]
```diff
@@ -96,6 +96,10 @@ jobs:
       # - name: Run cargo test - kata-cc (native-tls version) with keywrap-grpc + keywrap-jwe
       #   run: |
       #     sudo -E PATH=$PATH -s cargo test -p image-rs --no-default-features --features=encryption-openssl,keywrap-grpc,snapshot-overlayfs,signature-cosign-native,signature-simple,getresource,oci-distribution/native-tls,keywrap-jwe
+      
+      - name: Run cargo test - nydus
+        run: |
+          sudo -E PATH=$PATH -s cargo test -p image-rs --features nydus
```
Comment (fitzthum) [inline .github/workflows/image_rs_build.yml:102]: Do you need sudo here?

Comment (ChengyuZhu6) [reply .github/workflows/image_rs_build.yml:102]: This is consistent with the other test commands.

Comment (arronwy) [reply .github/workflows/image_rs_build.yml:102]: the mount operation in test cases need the root privilege

**Review by arronwy — APPROVED** [review]
Thanks @ChengyuZhu6 LGTM!

**Pull request merged**